cron/poll_pull_requests.py

```diff
@@ -95,6 +96,17 @@ def poll_pull_requests(api):
             # is our PR in voting window?
             in_window = gh.prs.is_pr_in_voting_window(api, pr, voting_window)
 
+            # Add the rights labels
+            __log.info("Adding PR %d labels regarding is patch", pr_num)
+
+            patched_files = gh.prs.get_patch(api, settings.URN, pr_num)
```

Comment:
You might want to try-catch this (and the following labeling code) just in case.